packages/mds-stream/nats/helpers.ts

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-import { connect, MsgCallback, SubscriptionOptions, Client } from 'ts-nats'
+import { connect, SubOpts, Msg, SubscriptionOptions, NatsConnection } from 'nats'
 import logger from '@mds-core/mds-logger'
 import { getEnvVar, asArray } from '@mds-core/mds-utils'
 import { SingleOrArray } from '@mds-core/mds-types'
 
 const initializeNatsClient = () => {
   const { NATS } = getEnvVar({ NATS: 'localhost' })
```

Comment:
I would love to see this renamed to `NATS_HOST` or `NATS_HOST/NATS_PORT` to be consistent with other services such as `KAFKA_`, `REDIS_`, `PG_`.

Reply:
So would I, but I'd rather that be in a targeted PR just for that change ^_^